jupyter-js-widgets/src/widget_int.js

```diff
@@ -200,23 +203,43 @@ var IntSliderView = widget.DOMWidgetView.extend({
             var readout = this.model.get('readout');
             if (readout) {
                 this.readout.style.display = '';
+                window.setTimeout(function() {
```

Comment:
Can you use requestAnimationFrame here? In order to batch calls, you can set a flag here to check the overflow, and then in the requestAnimationFrame, if the flag is set, do your thing and unset the flag. That way if this is triggered multiple times before an animation frame is callled, you only check things once.